using ExceptionSamples.BuiltIns;

Console.WriteLine("Start");
Person person = new();
person.CallMe(1);
Console.WriteLine("End");